core/domain/translation_domain.py

```diff
@@ -169,7 +162,7 @@ def __init__(
                 translated.
             content_format: TranslatableContentFormat. The format of the
                 content.
-            needs_update: bool. Whether the translated content needs update.
+            needs_update: bool. Whether the translation needs an update or not.
```

Comment:
I think the previous description is more clear because in the entire context we are referring it as translated content and not translation.

Reply:
Thanks for the catch, that seems reasonable. I'll update it!